Library/Formula/hspell.rb

```diff
@@ -0,0 +1,85 @@
+class Hspell < Formula
+  desc "The Hspell project is a free Hebrew linguistic project."
```

Comment:
You could shorten this into `"Free Hebrew linguistic project"`. `hspell` will be displayed before the description so I don’t think it’s necessary to repeat it.